### ISSUE TITLE
it seems that "period" granularity cannot be defined is always day

### ISSUE THREAD

**Issue by chicco785** [GoshPosh/grafana-meta-queries#22]
it would be nice to be able to use other granularities, such as hour, week, month or year

if this become a variable, the you can use the variable in the different expressions, e.g.:

```
 options.range.from._d = dateToMoment(options.range.from, false).add(periodsToShift,'days').toDate();
        options.range.to._d = dateToMoment(options.range.to, false).add(periodsToShift,'days').toDate();

```

looking at the moment lib, this look quite easy: https://momentjs.com/docs/#/manipulating/add/

**Comment by Gauravshah** [comment]
@chicco785 you are right, it should . not be very complicate the way code is written, can you go ahead an submit a PR. It will take me a while before I can get to it.

**Comment by Gauravshah** [comment]
fixed in #61 & #62
